uv/reporter/base.py

```diff
@@ -38,6 +38,20 @@ class AbstractReporter(metaclass=ABCMeta):
 
   """
 
+  def report_param(self, k: str, v: str) -> None:
```

Comment:
I'm adding these to the reporter interface. The default behavior is to do nothing.